scripts/postinstall.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-console */
+
+const { execSync, spawn } = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const deduplicate = require('yarn-deduplicate');
+
+if (process.env.NOYARNPOSTINSTALL) {
+  console.log('Skipping postinstall check');
+  process.exit(0);
+}
+
+process.chdir(path.resolve(__dirname, '..'));
+
+function syncWorkspace() {
```

Comment:
What problem does it solve?

Reply:
The idea was to make sure we specified the same version across the packages, i'm perfectly fine with removing it and just keeping the deduping. I don't mind at all